x-pack/plugins/cases/public/components/create/form_context.test.tsx

```diff
@@ -829,7 +608,10 @@ describe.skip('Create case', () => {
 
     await waitForComponentToUpdate();
```

Comment:
Maybe this call is now unnecessary. I noticed below you did something similar and replaced the 

`await waitForComponentToUpdate();` by a `await waitFor( ... toHaveBeenCalled()`

Reply:
you are right. It should be enough